features/step_definitions/active_docs_steps.rb

```diff
@@ -41,6 +41,28 @@
   assert_equal 1, evaluate_script("$('.apidocs-param-tips.#{autocomplete}:visible').length")
 end
 
+Then(/^the swagger v3 autocomplete should work for "(.*?)" with "(.*?)"$/) do |input_name, autocomplete|
+  id = 'default' # Could be passed as arg
+  section_id = "#operations-tag-#{id}"
+
+  closed_section = find("#{section_id}[data-is-open='false']")
+  closed_section&.click
```

Comment:
Why do we need this nil check? That smells like the test is not reliable

Reply:
When loading the Spec the methods are collapsed so that they have to be clicked. This is just in case they are loaded expanded already (in which case `closed_section` is `nil`). It's just a way to do `if closed then click it`. Would it be better this way?

Reply:
yes you can pass an options to `find` `visible: :all`

See https://rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Finders#find-instance_method

Reply:
Now that would make the test unreliable.. And we have had false positives in the past just because of that.

Reply:
I can remove the nil check and assume the section is collapse but I sincerely don't see any problem with checking if its closed first, in order to bypass a possible error.

Reply:
That's different, I did not mean to make the click but more:

- find the element is there (hidden or visible)
- if visible, click on it / because we want to close it, or the other way around right?
- if hidden, do nothing because it is what we want

Then do your stuff

It is more explicit what to find and more reliable in term of "elements are there, visible or not"

Reply:
It should always be collapsed at first (invisible) so it is always required to click on it. But just in case it someday reders expanded, do not click on it. It makes the cuke more robust IMHO. This doesn't even have to do anything with the assertion, it's just a mechanism to prepare the view in order to test it.

Reply:
Well when I read: `if` and `someday` that smells like not robust at all ... Uncertainty is worse than certainty
If it breaks in the future, let it break and fix it because we will know why and not just rely on supposition